PopMovies/PopMoviesTests/ValidateDecoderJSON/ValidadeDecodeJSON.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ValidadeDecoderJSON.swift
+//  PopMoviesTests
+//
+//  Created by Angela Alves on 19/10/22.
+//
+
+@testable import PopMovies
+import XCTest
+
+class ValidadeDecoderJSON: XCTestCase {
```

Comment:
Acho que rolou um typo aqui:
```suggestion
class ValidateDecoderJSON: XCTestCase {
```

Anyways, sugiro o nome `ValidateDataJSONDecoder`